src/styles/product.css

```diff
@@ -0,0 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body,
+p,
+h2 {
+  margin: 0;
+  color: var(--color_black);
+}
```

Comment:
`product.css` should only contain styles for an item that refers to `product`, these are about normalizing/resetting